core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.html

```diff
@@ -52,7 +52,7 @@
                             (skipQuestion)="skipCurrentQuestion()">
         </oppia-progress-nav>
       </div>
-      <div *ngIf="recommendedExplorationSummaries && recommendedExplorationSummaries.length > 0 && ((!isEndChapterCelebrationFeatureEnabled() && inStoryMode) || (collectionSummary && isOnTerminalCard() && isCurrentCardAtEndOfTranscript()))" class="conversation-skin-next-lesson-wrapper">
+      <div *ngIf="isOnTerminalCard() && recommendedExplorationSummaries && recommendedExplorationSummaries.length > 0 && ((!isEndChapterCelebrationFeatureEnabled() && inStoryMode) || (collectionSummary && isOnTerminalCard() && isCurrentCardAtEndOfTranscript()))" class="conversation-skin-next-lesson-wrapper">
```

Comment:
```suggestion
      <div *ngIf="isOnTerminalCard() && recommendedExplorationSummaries && recommendedExplorationSummaries.length > 0 && ((!isEndChapterCelebrationFeatureEnabled() && inStoryMode) || (collectionSummary && isOnTerminalCard() && isCurrentCardAtEndOfTranscript()))" 
           class="conversation-skin-next-lesson-wrapper">
```

Please write attributes on different lines.